packages/util/src/verkle.ts

```diff
@@ -188,12 +193,25 @@ export function getVerkleTreeIndexesForStorageSlot(storageKey: bigint): {
   return { treeIndex, subIndex }
 }
 
+/**
+ * Calculates the position of the code chunks in the Verkle tree, determining
+ * both the tree index (the node in the tree) and the subindex (the position within the node).
+ * @param {bigint} chunkId - The ID representing a specific chunk.
+ * @returns {Object} - An object containing:
+ */
 export function getVerkleTreeIndicesForCodeChunk(chunkId: number) {
   const treeIndex = Math.floor((VERKLE_CODE_OFFSET + chunkId) / VERKLE_NODE_WIDTH)
   const subIndex = (VERKLE_CODE_OFFSET + chunkId) % VERKLE_NODE_WIDTH
   return { treeIndex, subIndex }
 }
 
+/**
+ * Asynchronously calculates the Verkle tree key for the specified code chunk ID.
+ * @param {Address} address - The account address to access code for.
+ * @param {number} chunkId - The ID of the code chunk to retrieve.
+ * @param {VerkleCrypto} verkleCrypto - The cryptographic object used for Verkle-related operations.
```

Comment:
This is ideally how we refer to the verkle crypto object across our docs. FFI isn't really a helpful name for it in hindsight.